routes/registration.js

```diff
@@ -6,10 +6,25 @@ const models = require('../database/models');
 const utilityFunctions = require('./utilityfunctions');
 
 const localUser = models.user_account;
+const infokit = models.info_kit;
+const progress = models.progress;
 const validateEmail = utilityFunctions.validateEmail;
 const validateName = utilityFunctions.validateName;
 const validatePassword = utilityFunctions.validatePassword;
 const randomStr = utilityFunctions.randomString;
+
+function verificationMail(req, res, rString) {
+    mail.mailOptions.to = req.body.email;
+    mail.mailOptions.subject = 'PC PrepKit Email Verification';
+    mail.mailOptions.html = `<b>Click on the link to complete the verification</b> <a href='http://localhost:3000/verification?token=${rString}&user=${req.body.email}'>Verify</a>`;
```

Comment:
why are you hardcoding `localhost` in this? how will it work outside of local environment?